### ISSUE TITLE
Index out of range - power value conversion

### ISSUE THREAD

**Issue by jlmaners** [webdjoe/pyvesync#130]
I started receiving the following error message today after rebooting my Home Assistant instance.
```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 335, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/vesync/__init__.py", line 46, in async_setup_entry
    device_dict = await async_process_devices(hass, manager)
  File "/usr/src/homeassistant/homeassistant/components/vesync/common.py", line 19, in async_process_devices
    await hass.async_add_executor_job(manager.update)
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/pyvesync/vesync.py", line 296, in update
    device.update()
  File "/usr/local/lib/python3.9/site-packages/pyvesync/vesyncoutlet.py", line 69, in update
    self.get_details()
  File "/usr/local/lib/python3.9/site-packages/pyvesync/vesyncoutlet.py", line 175, in get_details
    power = round(float(Helpers.calculate_hex(power)), 2)
  File "/usr/local/lib/python3.9/site-packages/pyvesync/helpers.py", line 139, in calculate_hex
    converted_hex = (int(hex_conv[0], 16) + int(hex_conv[1], 16)) / 8192
IndexError: list index out of range
```
After pulling down the library and doing some of my own testing, it appears that the power values for my outlets are returning as integers or floats (depending on outlet type) and thus there isn't a hex string to split apart. In fact, it is only my offline entities that get the hex string because of the default value in the get request at 
https://github.com/webdjoe/pyvesync/blob/38bc617bd1572c5ef2b34c3758f91e268cfcb979/src/pyvesync/vesyncoutlet.py#L191

It appears that 7A outlets are returning as just 0 (don't have anything running to them currently), while my ESW15A and ESO15A appear to return float values (even when no power consumption is reported - ie 0.0).

Not being familiar with this code or previous behavior, I can't say if responses have changed in the API or if something else is going on. Please let me know if anymore information could be helpful.

**Comment by nstanley** [comment]
Having a similar issue on both my home assistant and wsl instance
```
Traceback (most recent call last):
  File "./vesync.py", line 5, in <module>
    manager.update()
  File "/home/nick/.local/lib/python3.8/site-packages/pyvesync/vesync.py", line 299, in update
    device.update()
  File "/home/nick/.local/lib/python3.8/site-packages/pyvesync/vesyncoutlet.py", line 86, in update
    self.get_details()
  File "/home/nick/.local/lib/python3.8/site-packages/pyvesync/vesyncoutlet.py", line 192, in get_details
    power = round(float(Helpers.calculate_hex(power)), 2)
  File "/home/nick/.local/lib/python3.8/site-packages/pyvesync/helpers.py", line 139, in calculate_hex
    converted_hex = (int(hex_conv[0], 16) + int(hex_conv[1], 16)) / 8192
IndexError: list index out of range
```
I know it worked this morning so maybe vesync updated their api?

**Comment by nprez83** [comment]
Same here, started having this error a couple of days ago, didn't really notice if it was before or after upgrading my home assistant core to 2022.4.5. I did try to go back to 2022.4.4 and it still wasn't working so much have broken before then.

**Comment by webdjoe** [comment]
Vesync changed the response from a hex to a numeric value. I'm surprised they left the endpoint the same. Just published a fix #131

**Comment by davidsat** [comment]
Thanks for the quick fix! I'm running supervised, and am wondering how to get the update. I'm under the impression this integration is part of the core release and I need to wait for a new release? If so, I'm wondering when the fix will be available for supervised users like me. Is there a 4.6.6 scheduled?
